Lib/typing.py

```diff
@@ -2722,10 +2722,9 @@ class Employee(NamedTuple):
     The resulting class has an extra __annotations__ attribute, giving a
     dict that maps field names to types.  (The field names are also in
     the _fields attribute, which is part of the namedtuple API.)
-    Alternative equivalent keyword syntax is also accepted::
-
-        Employee = NamedTuple('Employee', name=str, id=int)
+    An alternative equivalent functional syntax is also accepted::
 
+        Employee = NamedTuple('Employee', [('name', str), ('id', int)])
```

Comment:
This is a partial revert of #104945.

#104945 removed any mention of the functional syntax from this docstring. I agree with removing the mention of Python 3.5 from the docstring (also done in #104945), but I disagree with removing any mention of the functional syntax, since it is supported by type checkers, and still quite common in many codebases.

However, I _do_ think we should remove any mention of the keyword-argument syntax, since this way of creating `NamedTuple`s _isn't_ supported by any type checker that I know of.